src/views/connection.tsx

```diff
@@ -0,0 +1,26 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { Page } from './common'
+
+interface connection {
+  company_name: string
+  status: 'pending' | 'unverified' | 'verified_them' | 'verified_us' | 'verified_both' | 'disconnected'
+}
+
+@singleton()
+export default class ConnectionTemplates {
+  constructor() {}
+
+  public listPage = (connections: connection[]) => {
+    return (
+      <Page title="Veritable - Connections">
+        {connections.map((connection) => (
+          <div>
+            <div>{Html.escapeHtml(connection.company_name)}</div>
+            <div>{connection.status}</div>
+          </div>
+        ))}
+      </Page>
+    )
+  }
+}
```

Comment:
I guess each page will have their own templates. if I'm not mistaken CreateConnection, UpdatedConnection would go here, just to confirm: something like `notifications` we would create NotificationTemplates and relevant to the notification views/html/elements would be defined there?